src/components/List.js

```diff
@@ -11,6 +11,11 @@ const List = ({ results, setSearchTerm, searchTerm, token }) => {
   function checkTime(time) {
     return Date.now() - time <= 86400000; //number of milliseconds equal to 24 hours
   }
+  function handleDelete(result) {
+    let response = window.confirm(`Do you want to delete ${result.name}`);
```

Comment:
`window.confirm` is such an elegant solution over creating a custom modal! I think the copy could use a little more specificity, maybe "Permanently remove "`${results.name}` from your shopping list? You cannot undo this action, and your purchase history will be lost." so users fully understand the impact of the action.

Reply:
I like that. Its a bit more specific.

Reply:
Sounds good!  I can update it in a little bit!  We went with window.confirm for now because a lot of the modals are tied to specific styling libraries, so we figured we could do that once we decide how we're going to style the app.